### ISSUE TITLE
Writing to console is missing on second run of an application

### ISSUE THREAD

**Issue by k-rus** [Starcounter/Home#413]
I have following application for some testing purpose:

```c#
using Starcounter;
using System;
using Starcounter.ErrorCodes;

class Program {
    static void Main() {
        Db.SQL("create index ItemIndx on Item (ItemNr)");
		Db.Transact( delegate {
		var i1 = new Item { ItemNr = 1, Val = 2 };
		var i2 = new Item { ItemNr = 2, Val = 2 };
		var i3 = new Item { ItemNr = 3, Val = 2 };
		foreach( var i in Db.SQL<Item>("select i from item i where val = ?", 2)) {
			Console.WriteLine("Item "+ i.ItemNr + " having " + i.Val);
			if (i.ItemNr == 1)
				i2.Val = 4;
		}
		i1.Delete();
		i2.Delete();
		i3.Delete();
		});
        Db.SQL("drop index ItemIndx");
    }
}

[Database]
public class Item {
    public int ItemNr { get; set; }
	public int Val { get; set; }
}
```

When I run in empty database from command line, I see this output:
```bash
simpleTest -> default (started, default port 8080, admin 8181)                                                          
Item 1 having 2
Item 3 having 2
```
If I run it again, which restarts the app, the console is empty. The output after the both runs
```bash
>star ..\..\..\simpleTest.cs
simpleTest -> default (started, default port 8080, admin 8181)                                                          
Item 1 having 2
Item 3 having 2

>star ..\..\..\simpleTest.cs
simpleTest -> default (started, default port 8080, admin 8181)                                                          
```

If I stop the app, attach debugger, start the app, hit a break point and continue, then the output contains the expected two lines.

It looks like a bug.

I use local build on latest develop of L0 and L1

Can anyone reproduce it? Let me know if more information is needed.

@miyconst for planning.
I don't if @per-samuelsson can comment on this.

**Comment by per-samuelsson** [comment]
> I don't if @per-samuelsson can comment on this.

Nothing really helpful I'm afraid. I too have noticed from time to time random occasions where output routing don't seem to work properly, but never anything consistent. It's nothing new either, I've seen it for years I think.

If I remember correctly, it's  implemented now using some web-socket mechanism that writes from host to interested tools (like `star` and `staradmin`) and those write it to the console.

**Comment by miyconst** [comment]
I confirm the issue, just have experienced the same thing. Will see how to prioritize it.

**Comment by miyconst** [comment]
Closing as won't fix in Starcounter 2.4 and not applicable in Starcounter 3.0.
